creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -42,52 +38,7 @@ const FIND_SYNC_REQUESTS_JOB_MAX_LAST_SUCCESSFUL_RUN_DELAY_MS = config.get(
 const FIND_REPLICA_SET_UPDATES_JOB_MAX_LAST_SUCCESSFUL_RUN_DELAY_MS =
   config.get('findReplicaSetUpdatesJobLastSuccessfulRunDelayMs')
 
-// Helper Functions
-/**
- * Verifies that the request is made by the delegate Owner
- */
-const healthCheckVerifySignature = (req, res, next) => {
```

Comment:
is there a reason why we remove these health check helper fns?

Reply:
is it bc the ensureSPmiddleware does the same thing and we're using the ensureSPmiddleware in favor of this removed block?

Reply:
this was a hand rolled middleware specifically for these health check routes. the problem was the script needed the delegatePrivateKey of the node in order to run to run these health checks against the node. the change to ensureSPMiddleware allows any registered content node to run the checks